legacy/emergency-freeze/20250604_084529/continuity-states/mcp-continuity-service/manage-service.sh

```
@@ -0,0 +1,33 @@
+#!/bin/bash
+# MCP Continuity Service Manager
+
+case "$1" in
+  start)
+    echo "🚀 Starting services..."
+    ./start-service.sh
+    ;;
+  stop)
+    echo "🛑 Stopping services..."
+    if [ -f .service_pids ]; then
+      kill $(cat .service_pids) 2>/dev/null
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)